src/fielddef.ts

```diff
@@ -30,23 +30,37 @@ import {AggregatedFieldDef, WindowFieldDef} from './transform';
 import {getFullName, QUANTITATIVE, Type} from './type';
 import {contains, flatAccessWithDatum, getFirstDefined, replacePathInField, titlecase} from './util';
 
-type Value = number | string | boolean | null;
+type VisualValue = number | string;
```

Comment:
1) value can be null too, why are you removing it? 

2) Better not rename and keep it `Value`